sfdoc/publish/tests/test_integration.py

```diff
@@ -40,7 +45,11 @@ def integration_test(func):
     )(func)
 
 
-class SFDocTestIntegration(TestCase):
+class TstHelpers:
```

Comment:
Is the missing e so that this doesn't get collected by the testrunner? If so, probably worth a comment.

Reply:
4c1cae9b9daa1d876f803a4658a1eb04db8e40a7